article/_posts/2020-11-20-FaderNetworks.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: review
+title: "Fader Networks: Manipulating Images by Sliding Attributes"
+tags: deep-learning autoencoder
+author: "Nathan Painchaud"
+cite:
+    authors: "Guillaume Lample, Neil Zeghidour, Nicolas Usunier, Antoine Bordes, Ludovic Denoyer, Marc'Aurelio Ranzato"
+    title:   "Fader Networks: Manipulating Images by Sliding Attributes"
+    venue:   "NeurIPS (2017)"
+pdf: "https://arxiv.org/pdf/1706.00409.pdf"
+---
+
+
+# Highlights
+- New encoder-decoder architecture that disentangle salient information in images and attributes directly in the latent
+space;
+- At test-time, using continuous attribute values allows to adjust how much of an attribute is perceivable in the
+generated image.
+
+
+# Introduction
+The authors' goal is to control some attributes of interest images, for which transformations are ill-defined and
+training is unsupervised, i.e. no examples of images before and after the transformations are available. They achieve
+this by learning a latent space that is invariant to the attributes of interest, which can then simply be concatenated
+to any latent vector during the generative process.
+
+
+# Methods
+The attribute-invariance is obtained through a process similar to domain-adversarial training, where a classifier learns
+to predict attributes $$y$$ from the latent representation $$z$$, while the encoder-decoder tries to simultaneously
+reconstruct the input and fool the classifier (c.f. Figure 2).
+
+![](/article/images/FaderNetworks/figure2.jpg)
+
+## Discriminator objective
+The discriminator tries to predict the attributes of the input image given its latent representation according to:
+
+$$
+\mathcal{L}_{\text{dis}}(\theta_{\text{dis}}|\theta_{\text{enc}}) = 
+-\frac{1}{m} \sum_{(x,y) \in D} \text{log} P_{\theta_{\text{dis}}}(y | E_{\theta{\text{enc}}}(x))
+$$
+
+## Adversarial objective
+The encoder tries to learn that optimizes the two objectives mentioned above: namely a good reconstruction but a bad
```

Comment:
something is missing here: "(...) to learn that (...)" or maybe "that" is not necessary.